Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditquerylog/nhinc/proxy/AuditQueryLogProxyWebServiceUnsecuredImpl.java

```diff
@@ -0,0 +1,172 @@
+    @Override
+    public QueryAuditEventsResponseType retrieveAudits(QueryAuditEventsRequestType request) {
+
+        QueryAuditEventsResponseType response = null;
+        try {
+
+            CONNECTClient<AuditQueryLogPortType> client = getCONNECTClient(getQueryAuditEventsPortDescriptor());
+            response = (QueryAuditEventsResponseType) client.invokePort(AuditQueryLogPortType.class,
+                QUERY_AUDIT_EVENTS, request);
+
+        } catch (Exception ex) {
+            LOG.error("Failed to call the web service " + NhincConstants.ADAPTER_AUDIT_QUERY_LOG_SERVICE_NAME
+                + ex.getLocalizedMessage(), ex);
+        }
+        return response;
+    }
+
+    /**
+     *
+     * @param request - Request provides search params MessageId and RelatesTo to retrieve Audit Events. If none of the
+     * elements are provided in request. If optional elements are not provided will return all audit events
+     * @return QueryAuditEventsResponseType
+     */
+    @Override
+    public QueryAuditEventsResponseType retrieveAuditsByMsgIdAndRelatesToId(
+        QueryAuditEventsRequestByRequestMessageId request) {
+        QueryAuditEventsResponseType response = null;
+        try {
+
+            CONNECTClient<AuditQueryLogPortType> client = getCONNECTClient(
+                getQueryAuditEventsByMessageIdAndRelatesToPortDescriptor());
+            response = (QueryAuditEventsResponseType) client.invokePort(AuditQueryLogPortType.class,
+                QUERY_AUDIT_EVENTS_BY_MSG_RELATESTO_ID, request);
+
+        } catch (Exception ex) {
+            LOG.error("Failed to call the web service " + NhincConstants.ADAPTER_AUDIT_QUERY_LOG_SERVICE_NAME
+                + ex.getLocalizedMessage(), ex);
+        }
+        return response;
+    }
+
+    /**
+     *
+     * @param request - Request provides Id and corresponding Blob will be retrieved.
+     * @return
+     */
+    @Override
+    public QueryAuditEventsBlobResponse retrieveAuditBlob(QueryAuditEventsBlobRequest request) {
+        QueryAuditEventsBlobResponse response = null;
+        try {
+
+            CONNECTClient<AuditQueryLogPortType> client = getCONNECTClient(getQueryAuditEventsByIdPortDescriptor());
+            response = (QueryAuditEventsBlobResponse) client.invokePort(AuditQueryLogPortType.class,
+                QUERY_AUDIT_EVENTS_BY_ID, request);
+
+        } catch (Exception ex) {
+            LOG.error("Failed to call the web service " + NhincConstants.ADAPTER_AUDIT_QUERY_LOG_SERVICE_NAME
+                + ex.getLocalizedMessage(), ex);
+        }
+        return response;
+    }
+
+    private String getAdapterQueryLogUnsecuredSerrviceUrl() {
+        String url = null;
+        try {
+            url = proxyHelper.getUrlLocalHomeCommunity(NhincConstants.ADAPTER_AUDIT_QUERY_LOG_SERVICE_NAME);
+        } catch (ConnectionManagerException ex) {
+            LOG.debug("Error while retrieving url for " + NhincConstants.ADAPTER_AUDIT_QUERY_LOG_SERVICE_NAME
```

Comment:
Instead of checking if log levels are enabled, we should take advantage of our recent SLF4J migration and use parameterized logging statements going forward.  Definitely something we should discuss as a team.